### PR TITLE
Rewrote to almost one block of array initialization

### PR DESCRIPTION
**Pull request by Quix0r**
Changed:
- rewrote to almost one block of array initialization
- followup PR to #11763

### CODE REVIEW

**Comment by MrPetovan** [comment]
Thanks, but this is a different file, it won't affect https://github.com/friendica/friendica/pull/11763, when I said "change request" I meant submitting a review with a suggested change in a review comment.

File: src/Model/Event.php [hunk 1]
```diff
@@ -243,27 +243,29 @@ public static function delete(int $event_id)
...
+			'deny_gid'  =>        $arr['deny_gid']  ?? '',
+			'nofinish'  => intval($arr['nofinish'] ?? (!empty($arr['start']) && empty($arr['finish']))),
+			'created'   => DateTimeFormat::utc(($arr['created'] ?? '') ?: 'now'),
+			'edited'    => DateTimeFormat::utc(($arr['edited']  ?? '') ?: 'now'),
+			'start'     => DateTimeFormat::utc(($arr['start']   ?? '') ?: DBA::NULL_DATETIME),
+			'finish'    => DateTimeFormat::utc(($arr['finish']  ?? '') ?: DBA::NULL_DATETIME),
+		];
+
+		$event['uri']    = ($arr['uri']       ?? '') ?: Item::newURI($event['guid']);
+		$event['uri-id'] = ItemURI::insert(['uri' => $event['uri'], 'guid' => $event['guid']]);
```
Comment (MrPetovan) [inline src/Model/Event.php]: Please initialize a `$guid` and `$uri` variables before the array block, this way you don't have the `$event` array to reference itself.
```suggestion
		$guid = $arr['guid'] ?? '' ?: System::createUUID();
		$uri  = $arr['uri']  ?? '' ?: Item::newURI($guid);
```

**Review by MrPetovan — CHANGES_REQUESTED** [review]

File: src/Model/Event.php (hunk 1 above)
Comment (Quix0r) [reply src/Model/Event.php]: If `ItemURI::insert()` fails, a `false` might be included in `$event`. I guess I should better rewrite this?

**Review by MrPetovan — APPROVED** [review]

File: src/Model/Event.php (hunk 1 above)
Comment (MrPetovan) [reply src/Model/Event.php]: It's fine for now.

**Pull request merged**